website/src/pages/terms-of-service.tsx

```diff
@@ -189,4 +190,10 @@ const TermsOfService = () => {
 
 TermsOfService.getLayout = getTransparentHeaderLayout;
 
+export const getStaticProps = async ({ locale }) => ({
+  props: {
+    ...(await serverSideTranslations(locale, ["common"])),
+  },
+});
+
```

Comment:
can you extract this into a function?

Reply:
I agree, ideally this is a little simpler so that the same 6 lines aren't copied for every page.

Reply:
Thanks for the review, I agree with you.  I referenced the instructions from `next-i18next` in the PR description which advises to add this to each page level component.  I  have refactored this to a export function which is only one line now for default namespace and common static props, please take a look. 

See also this related issue: https://github.com/i18next/next-i18next/issues/1215